sbat.c

```diff
@@ -394,33 +404,29 @@ set_sbat_uefi_variable(void)
 			case SBAT_POLICY_LATEST:
 				dprint("Custom sbat policy: latest\n");
 				sbat_var = SBAT_VAR_LATEST;
+				clear_sbat_policy();
 				break;
 			case SBAT_POLICY_PREVIOUS:
 				dprint("Custom sbat policy: previous\n");
 				sbat_var = SBAT_VAR_PREVIOUS;
 				break;
 			case SBAT_POLICY_RESET:
 				if (secure_mode()) {
-					console_print(L"Cannot reset SBAT policy: Secure Boot is enabled.\n");
+					console_print(L"Cannot reset SBAT policy: "
```

Comment:
Don't break strings across lines for formatting, leave them intact so they're easy to grep for.